starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -50,3 +50,11 @@ public void findAll() {
     Assert.assertFalse(actualInventory.isEmpty());
   }
 }
+//task1
```

Comment:
This comment should be similar to other test comments for style/consistent documentation.
e.g.,: Replace the below comment with a more relevant description
 /**
   * Test Find All method.
   */